src/containers/MainLayout.tsx

```diff
@@ -47,6 +55,28 @@ const MainLayout: React.FC<MainLayoutProps> = ({ children }) => {
   const theme = useMantineTheme()
   const { sidebarOpen, toggleSidebar } = useSidebar()
   const { classes } = useStyles()
+  //Randomize logo logic:
+  const logoes = [
+    logoUrl1,
+    logoUrl2,
+    logoUrl3,
+    logoUrl4,
+    logoUrl5,
+    logoUrl6,
+    logoUrl7,
+  ]
+  function randomizeLogo() {
+    const index = Math.floor(Math.random() * logoes.length)
+    return logoes[index]
+  }
+  const [logoUrl, setLogoUrl] = React.useState(logoes[0])
+  React.useEffect(() => {
+    const intervalId = setInterval(() => {
+      setLogoUrl(randomizeLogo()) // Update logoUrl every 5 seconds
+    }, 5000) // 5000 milliseconds = 5 seconds
+
+    return () => clearInterval(intervalId) // Cleanup the interval on component unmount
```

Comment:
Comment is redundant

```suggestion
    return () => clearInterval(intervalId)
```